user_service/tilt/user_service_tilt.py

```diff
@@ -0,0 +1,70 @@
+from tilt import tilt
+import json
+from hashlib import sha256
+from datetime import datetime
+
+#List of legal_bases with reference and description
+first_legal_base = tilt.AnyOfSchemaForTheLegalBasesOfTheDataDisclosed(
+    reference="GDPR-99-1-a",
+    description="General Data Protection Regulation (GDPR) Art."
+)
+second_legal_base = tilt.AnyOfSchemaForTheLegalBasesOfTheDataDisclosed(
+    reference="BDSG-42-5",
+    description="BDSG-42-5 refers to the processing of personal data within.."
+)
+
+#Store the user-service-tilt
+temporal = tilt.TemporalElement(
+    description='Temporal element for the user-service-tilt',
+    ttl=''
+)
+storage = tilt.StorageElement(
+    temporal=[temporal], 
+    purpose_conditional=["Data is always stored"],
+    legal_basis_conditional=["SGB-100-42"],
+    aggregation_function=tilt.AggregationFunction.MIN
+    )
+
+legitimate_interests = tilt.AnyOfSchemaForLegitimateInterests(
+    exists=False,
+    reasoning="No legitimate interests"
+)
+
+non_disclosure = tilt.NonDisclosure(
+    legal_requirement=False,
+    contractual_regulation=False,
+    obligation_to_provide=False,
+    consequences="If the data is not disclosed, the shipment cannot be delivered."
+)
+
+purposes = tilt.AnyOfSchemaForThePurposes( 
+    purpose="To provide the user-service-tilt",
+    description="To provide the user-service-tilt"
+)
+
+first_recipient = tilt.Recipient(
+    name="",
+    address="",
+    category="",
+    country="DE",
+    division="",
+    representative=None
+)
+
+#Data disclosed to user-service-tilt
+disclosed_data = tilt.DataDisclosedElement(
+    id='user-service-tilt-01',
+    category="Language Preference",
+    legal_bases=[first_legal_base, second_legal_base],
+    storage=[storage],
+    legitimate_interests=[legitimate_interests],
+    non_disclosure=non_disclosure,
+    purposes=[purposes],
+    recipients=[first_recipient]
+)
+
+tilt_dict = {}
+tilt_dict['dataDisclosed'] = disclosed_data.to_dict()
```

Comment:
```suggestion
tilt_dict['dataDisclosed'] = [disclosed_data.to_dict()]
```
dataDisclosed is supposed to be an array.

Reply:
Yeap of course sorry, I missed it!